source/globus/globus_platform.rst

```diff
@@ -0,0 +1,70 @@
+.. raw:: html
+
+   <style> .bgrnd1 {background-color:#DCDCDC; border-radius:5px} </style>
+
+.. role:: bgrnd1
+
+
+.. _globus_platform:
+
+Globus file and data sharing platform
+=====================================
+
+The Globus platform enables developers to provide robust file transfer, sharing and search capabilities within their own research data applications and services, while leveraging advanced identity management, single sign-on, and authorization capabilities.
+
+This document is a hands-on guide to the Globus file sharing platform. It should complement the official documentation at `docs.globus.org <https://docs.globus.org>`_.
+
+For all questions concerning the Globus file sharing platform, please contact the VSC Globus team at data@vscentrum.be.
+
+.. include:: globus_contact_information.rst
+
+We welcome your feedback, comments and suggestions for improving the Globus Tutorial.
+
+
+.. _access:
+
+Access
+------
+
+.. include:: access.rst
+
+
+.. _managing_and_transferring_files:
+
+Managing and transferring files
+-------------------------------
+
+.. include:: managing_and_transferring_files.rst
+
+
+.. local_endpoints
+
+Local endpoints
```

Comment:
Would it make sense to first talk about local endpoints, and only then about file sharing?  The advantage would be that people can follow along.

Reply:
Maybe. Until the next user asks the opposite due to the interrelated nature of both.

Reply:
I'm not questioning it is technically possible and already implemented.  I'm just wondering whether it would make sense to add an explicit remark in the Globus docs that states that a very nice use case is data transfer between Tier-1 and Tier-2 across sites.  It is typically something users struggle with, and it would motivate them to start using Globus.